.circleci/config.yml

```diff
@@ -3,25 +3,61 @@
 # Check https://circleci.com/docs/2.0/language-ruby/ for more details
 #
 version: 2.1
-jobs:
-  build:
+
+executors:
+  ruby_browsers:
     docker:
-      # Specify the Ruby version you desire here
-      - image: cimg/ruby:3.0.5
+      - image: cimg/ruby:3.0.4-browsers
+        environment:
+          BUNDLER_VERSION: 2.3.13
 
-    working_directory: ~/identity-saml-sinatra
+commands:
+  bundle-yarn-install:
```

Comment:
should we change this name to `install-dependencies` since it's doing the bundle as well?